homeassistant/components/kraken/strings.json

```diff
@@ -0,0 +1,26 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "[%key:common::config_flow::abort::single_instance_allowed%]"
+        },
+        "error": {},
+        "step": {
+            "user": {
+                "data": {},
+                "description": "[%key:common::config_flow::description::confirm_setup%]",
+                "title": "Kraken.com"
```

Comment:
No need to add, it defaults to integration name.

Reply:
Thanks!